### PR TITLE
feat: add layers, renderers, useConstraints documentation is added.

### PR DESCRIPTION
**Pull request by Kate-Lyndegaard**
Added documentation updates based on the last deploy: ability to add layers to the layer widget in the map preview, configuring renderers in the Feature Types section of a theme and clarification on adding additional otherConstraints elements to metadata. Fixed some markdown typos.

### CODE REVIEW

File: docs/create-manage-datasets/map-preview/2014-01-02-map-preview.md [hunk 1]
```diff
@@ -16,24 +16,28 @@ The map preview is a simple web map application that enables the viewing of publ
 
 The layer widget contains:
 
-    * Legend Information,
-    * the options to turn layers on and off, and
-    * the option to filter raster datasets for display within a dataset series
+* Legend Information,
```
Comment (JohannaOtt) [inline docs/create-manage-datasets/map-preview/2014-01-02-map-preview.md:19]: Why are the L and I upper case here and the next bullet points not?

File: docs/setup-hc/create-a-theme/2015-02-01-theme-edit-view-config.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ language: en
 ---
 
 
-A common issue with type and property names from data schemas is that they are hard to understand. They tend to be cryptic abbreviations, especially when they come from a Shapefile, or are defined using a foreign language. To provide effective view services despite this, you can define display properties which are then used for legends and GetFeatureInfo popups.
+A common issue with type and property names from data schemas is that they are hard to understand. They tend to be cryptic abbreviations, especially when they come from a Shapefile, or are defined using a foreign language. To provide effective view services despite this, you can define display properties which are then used for legends and GetFeatureInfo popups. hale»connect supports the configuration aliases for feature type names and feature properties.
```
Comment (JohannaOtt) [inline docs/setup-hc/create-a-theme/2015-02-01-theme-edit-view-config.md:12]: I wonder if we should keep the first two sentences here. To me, it sounds pretty much like our opinion and nothing really relevant for the product documentation. What do you think?

configuration aliases -> configuration of aliases

File: docs/setup-hc/create-a-theme/2015-02-01-theme-edit-view-config.md [hunk 3]
```diff
@@ -19,3 +19,19 @@ To set display properties, follow these steps as a logged in theme manager or or
...
 6.	Save your configuration by clicking on the &laquo;Save&raquo; button at the end of the table.
+
+# Formatting display values #
+
+The renderer enables users to control the formatting of values displayed in the Feature Explorer. Users have the option to select one of the following renderers:
+
+* Default (no renderer)
+* Formatted number (enter a number format, e.g. ###.##)
+* Formatted Date (choose a date format option)
+* Codelist Labels (substitute coded values can be provided in lookup table)
```
Comment (JohannaOtt) [inline docs/setup-hc/create-a-theme/2015-02-01-theme-edit-view-config.md:30]: Why is the L uppercase?

File: docs/setup-hc/edit-metadata-config/2015-02-10-theme-edit-metadata.md [hunk 4]
```diff
@@ -454,6 +454,45 @@ The ``defaultValue`` field can be populated with free text, provided in double q
                 }
             }
 
+### Adding more than one ```otherConstraints``` element
```
Comment (JohannaOtt) [inline docs/setup-hc/edit-metadata-config/2015-02-10-theme-edit-metadata.md:457]: Isn't that just the same as for all other elements having cardinality max > 1 and where enumeration values are used for defaultValue or autofillRule as described [here](https://help.wetransform.to/docs/setup-hc/edit-metadata-config/2015-02-10-theme-edit-metadata#adding-multiple-default-values-for-one-metadata-field)?
Or am I missing something?

File: i18n/de/docusaurus-plugin-content-docs/current/create-manage-datasets/map-preview/2014-01-02-map-preview.md [hunk 5]
```diff
@@ -16,9 +16,9 @@ Die Kartenansicht ist eine einfache Web-Anwendung, die die Anzeige von veröffen
 
 Das Layer-Widget enthält
 
-    * Legendeninformationen,
-    * die Option zum Ein- und Ausschalten von Layern und
-    * die Option zum Filtern von Raster-Datensätzen für die Anzeige innerhalb einer Datensatzserie.
+* Legendeninformationen,
+* die Option zum Ein- und Ausschalten von Layern und
+* die Option zum Filtern von Raster-Datensätzen für die Anzeige innerhalb einer Datensatzserie.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/create-manage-datasets/map-preview/2014-01-02-map-preview.md:21]: Is "die Option, externe WMS-Layer hinzuzufügen" missing?

**Review by JohannaOtt — APPROVED** [review]

File: docs/setup-hc/edit-metadata-config/2015-02-10-theme-edit-metadata.md (hunk 4 above)
Comment (Kate-Lyndegaard) [reply docs/setup-hc/edit-metadata-config/2015-02-10-theme-edit-metadata.md:457]: @JohannaOtt Yes, it's the same. The issue is that the default hc metadata template includes the DE open data licence in a different section. Andreas added this a long time ago. Some customers get confused when they see the licence they need to add in a different section of the metadata than where the new GDI-DE Metadata Konventionen requires it. I will update the existing description.

File: i18n/de/docusaurus-plugin-content-docs/current/create-manage-datasets/map-preview/2014-01-02-map-preview.md (hunk 5 above)
Comment (Kate-Lyndegaard) [reply i18n/de/docusaurus-plugin-content-docs/current/create-manage-datasets/map-preview/2014-01-02-map-preview.md:21]: I will add the DE translations now.

**Pull request merged**